src/systems/user_commands/UserCommands.cc

```diff
@@ -331,6 +333,36 @@ class VisualCommand : public UserCommandBase
                   aMaterial.emissive().a(), bMaterial.emissive().a(), 1e-6f);
             }};
 };
+
+/// \brief Command to modify a wheel entity from simulation.
+class WheelSlipCommand : public UserCommandBase
+{
+  /// \brief Constructor
+  /// \param[in] _msg Message containing the wheel slip parameters.
+  /// \param[in] _iface Pointer to user commands interface.
+  public: WheelSlipCommand(msgs::WheelSlipParameters *_msg,
+      std::shared_ptr<UserCommandsInterface> &_iface);
+
+  // Documentation inherited
+  public: bool Execute() final;
+
+  /// \brief WheelSlip equality comparision function
+  public: std::function<bool(const msgs::WheelSlipParameters &, const msgs::WheelSlipParameters &)>
+          wheelSlipEql {
+            [](const msgs::WheelSlipParameters &_a, const msgs::WheelSlipParameters &_b) {
+              return
+                _a.name() == _b.name() &&
+                _a.id() == _b.id() &&
```

Comment:
do we need to compare the `parent_name` and `parent_id` fields also?

Reply:
```suggestion
                _a.id() == _b.id() &&
                _a.parent_id() == _b.parent_id() &&
                _a.parent_name() == _b.parent_name() &&
```

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/1241/commits/906c185084e9ca8970a1dc290fef7c0cdcefd1c4